Dockerfile.webhook

```diff
@@ -8,6 +8,6 @@ RUN rm -rf /go/src/github.com/openshift/csi-driver-shared-resource/examples
 RUN rm -f /go/src/github.com/openshift/csi-driver-shared-resource/vendor/k8s.io/apimachinery/pkg/util/managedfields/pod.yaml
 RUN make build-webhook
 
-FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+FROM registry.access.redhat.com/ubi9:9.3
```

Comment:
Can't we use the ubi9-minimal?

Reply:
Yes, this should be minimal, let me replace.